apps/www/src/app/notes/components/form.tsx

```diff
@@ -103,6 +110,25 @@ export function NoteForm({ user, currentNote }: Props) {
     logEvent(analytics, "update_note");
   };
 
+  // Dynamic Textarea Height
```

Comment:
can we convert the dynamic textarea height into a custom hook? since we're also using it in other places with the same implementation